### ISSUE TITLE
How is linking to the other github repo supposed to work?

### ISSUE THREAD

**Issue by JohnMcLear** [manuels/ep_latex#3]
I'm confused..  Is that even possible?  npm install will just install whatever files are in the plugin folder, afaik any symbolic git links will break in transit?

**Comment by manuels** [comment]
hm, looks like we have to add a repo section to package.json

```
  "repository": {
    "type": "git",
    "url": "..."
  },
```

That should do the trick

**Comment by JohnMcLear** [comment]
I'm not sure it does..  I think that just provides the developer with a link to the repo.  I don't think it includes the repo itself..  I'm not sure this is actually possible

**Comment by manuels** [comment]
ok, there seems to be a way to make it work: http://stackoverflow.com/questions/7598814/how-can-i-handle-node-js-dependencies-in-a-project-on-git-with-npm

**Comment by JohnMcLear** [comment]
Are your packages available via npm?  I don't see it here.. https://encrypted.google.com/search?q=texlive&q=site:npmjs.org&hl=en

**Comment by manuels** [comment]
no, I just updated the code in this repo. It's nearly working. I just cannot figure out the url in etherpad-lite of the stuff I put into static/js/

**Comment by JohnMcLear** [comment]
oh that's prolly /static/plugins/ep_latex/static/js :)

**Comment by manuels** [comment]
this plugin should be available as 'ep_latex' in the npm.
